develop/plone/sessions/cookies.rst

```diff
@@ -2,71 +2,73 @@
 Cookies
 =======
 
-.. admonition:: Description
+.. topic:: Description
 
-    Handling session and other cookies in Plone
+    Handling cookies in Plone
 
 
 Introduction
 ============
 
-Setting and getting cookies
+Cookies in Plone are read from the request and set on the response.
+There is an API for cookie handling.
```

Comment:
Where is the API? Got link, docstrings, or Sphinx autodoc?

Reply:
Ok, you're right, API is too much for 3 methods... I'll rephrase this.